sklearn/utils/estimator_checks.py

```diff
@@ -67,7 +67,17 @@
                 'RandomForestRegressor', 'Ridge', 'RidgeCV']
 
 
-def _yield_non_meta_checks(name, Estimator):
+def assert_almost_equal_dense_sparse(x, y, decimal=6, err_msg=''):
+    if sparse.issparse(x):
+        assert_array_almost_equal(x.data, y.data,
```

Comment:
name's shorter? (i.e. not obscenely verbose) :D

More seriously, because it allows control in terms of absolute and relative tolerance, which allows you to compare very large numbers, very small numbers, etc., while decimals is irrelevant to the internal number format.